### PR TITLE
Questions to Kaggle dataset

### PR DESCRIPTION
**Pull request by Makhsuda**

### CODE REVIEW

**Comment by chendaniely** [comment]
@Makhsuda can you fix the`[kaggle]` so it also has the link to the [task](https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge/tasks) with `[kaggle](url)`

was this the task you were referring to? https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge/tasks?taskId=572

File: analysis/db/max/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+I think that the following questions might also be analyzed using available dataset, starting with titles and abstracts of the articles.
+
+-What sources, best practices or challenges are in medical care to prevent spreading
+the SARS-CoV-2? (There was task in [Kaggle] as well)
+
+-What are the changes in people's behavior during this COVID-19 pandemic period?
```
Comment (chendaniely) [inline analysis/db/max/README.md:6]: Where do you plan on getting (text?) data to look at "behavior"?

File: analysis/db/max/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+I think that the following questions might also be analyzed using available dataset, starting with titles and abstracts of the articles.
+
+-What sources, best practices or challenges are in medical care to prevent spreading
```
Comment (ambrown77) [inline analysis/db/max/README.md:3]: how will you define a best practice or a challenge in the data pull?

File: analysis/db/max/README.md (hunk 1 above)
Comment (Makhsuda) [reply analysis/db/max/README.md:6]: I think it is possible to get from articles, like maybe what products are causing higher demand?
In my opinion, we can also see how offices/universities/other organizations react to this situation from their announcements. Besides, what platforms are causing more demand like zoom (I have recently read an article how Zoom has the highest stock market price), which means that more people started working/studying from home. Also, I am not sure if it's possible to get some data from posts in social media.

File: analysis/db/max/README.md (hunk 2 above)
Comment (Makhsuda) [reply analysis/db/max/README.md:3]: Maybe through the keywords like lack of smth (tests), invention or some other related words? It seems that it is not the most efficient way to do through keywords, but I haven't found another alternative yet.

**Pull request merged**